src/uint.rs

```diff
@@ -2148,18 +2142,18 @@ mod tests {
 
 	#[test]
 	fn leading_zeros() {
-		assert_eq!(U256::from("000000000000000000000001adbdd6bd6ff027485484b97f8a6a4c7129756dd1").leading_zeros(), 95);
-		assert_eq!(U256::from("f00000000000000000000001adbdd6bd6ff027485484b97f8a6a4c7129756dd1").leading_zeros(), 0);
-		assert_eq!(U256::from("0000000000000000000000000000000000000000000000000000000000000001").leading_zeros(), 255);
-		assert_eq!(U256::from("0000000000000000000000000000000000000000000000000000000000000000").leading_zeros(), 256);
+		assert_eq!("000000000000000000000001adbdd6bd6ff027485484b97f8a6a4c7129756dd1".parse::<U256>().unwrap().leading_zeros(), 95);
```

Comment:
why would you change this? imo it's less readable

Reply:
Because `into` shouldn't panic. Current implementation cannot convert any string into U256. This is what `parse` is for.